ui/app/components/send_/send.utils.js

```diff
@@ -186,3 +209,27 @@ function generateTokenTransferData (selectedAddress, selectedToken) {
     x => ('00' + x.toString(16)).slice(-2)
   ).join('')
 }
+
+function hexComparator (a, b) {
+  return conversionGreaterThan(
+    { value: a, fromNumericBase: 'hex' },
+    { value: b, fromNumericBase: 'hex' },
+  ) ? 1 : -1
+}
+
+function estimateGasPriceFromRecentBlocks (recentBlocks) {
+  // Return 1 gwei if no blocks have been observed:
+  if (!recentBlocks || recentBlocks.length === 0) {
+    return ONE_GWEI_IN_WEI_HEX
+  }
+  const lowestPrices = recentBlocks.map((block) => {
+    if (!block.gasPrices || block.gasPrices.length < 1) {
+      return ONE_GWEI_IN_WEI_HEX
+    }
+    return block.gasPrices
+      .sort(hexComparator)[0]
```

Comment:
Might be better to use something like `reduce` to get the lowest value rather than `sort` since it would require just one pass through the list.

Reply:
done in e042c13